apps/core/serializers/register.py

```diff
@@ -35,7 +35,9 @@ def create(self, validated_data):
             preference_model.objects.create(user=user)
 
         try:
-            helpers.send_verification_email(user, url=settings.VERIFY_EMAIL_URL)
+            helpers.send_verification_email(
+                user, url=settings.VERIFY_EMAIL_URL, email=validated_data["email"].lower()
```

Comment:
You have same email reference on line 26, it should probably be a new declared variable right now